tests/unit/hive_metastore/test_tables.py

```diff
@@ -76,7 +76,7 @@ def test_sql_managed_non_delta():
                 catalog="catalog",
                 database="db",
                 name="managed_table",
-                object_type="MANAGED",
+                object_type="EXTERNAL",
```

Comment:
can you add new unit tests instead of changing existing ones? this risks at introducing bugs. we didn't change any logic - we've added new logic